python-backend/app/api/parties/party/models/party.py

```diff
@@ -15,15 +15,22 @@
 class Party(AuditMixin, Base):
     __tablename__ = 'party'
     party_guid = db.Column(UUID(as_uuid=True), primary_key=True)
-    first_name = db.Column(db.String(100), nullable=True)
-    middle_name = db.Column(db.String(100), nullable=True)
-    party_name = db.Column(db.String(100), nullable=False)
-    phone_no = db.Column(db.String(10), nullable=False)
-    phone_ext = db.Column(db.String(4), nullable=True)
-    email = db.Column(db.String(254), nullable=False)
+    first_name = db.Column(db.String, nullable=True)
+    middle_name = db.Column(db.String, nullable=True)
+    party_name = db.Column(db.String, nullable=False)
+    phone_no = db.Column(db.String, nullable=False)
+    phone_ext = db.Column(db.String, nullable=True)
+    email = db.Column(db.String, nullable=False)
     effective_date = db.Column(db.DateTime, nullable=False, default=datetime.utcnow)
     expiry_date = db.Column(db.DateTime, nullable=False, default=datetime.strptime('9999-12-31', '%Y-%m-%d'))
-    party_type_code = db.Column(db.String(3), db.ForeignKey('party_type_code.party_type_code'))
+    party_type_code = db.Column(db.String, db.ForeignKey('party_type_code.party_type_code'))
+
+    suite_no = db.Column(db.String, nullable=True)
+    address_line_1 = db.Column(db.String, nullable=True)
+    address_line_2 = db.Column(db.String, nullable=True)
+    city = db.Column(db.String, nullable=True)
+    province_code = db.Column(db.String, nullable=True)
```

Comment:
At once point support for international addresses was raised, as companies possibly have international contacts. May want to validate that that is not in scope. I think there's a standard set of fields for international addresses (4 address lines, locale, region, zip, country or along that line).

Reply:
Yep, we discussed this as well before I moved ahead on implementing this. Kristin said to stick with province_code and postal_code at this time.

It may be worth re-addressing this if you think it could lead to breaking API changes.

Reply:
Seems short sighted not to nail this down. If we intend to support it in the future we use those fields today. I don't think its extra complexity, and if we want to show "province" in our UI and have more generic fields in our DB we can do that. @klefler @itchison thoughts?

https://stackoverflow.com/questions/929684/is-there-common-street-addresses-database-design-for-all-addresses-of-the-world

    Address Lines 1-4
    Locality
    Region
    Postcode (or zipcode)
    Country


https://stackoverflow.com/questions/126207/best-practices-for-consistent-and-comprehensive-address-storage-in-a-database
    name,
    company-name,
    street-lines[], // up to 4 free-type street lines
    county/sublocality,
    city/town/district,
    state/province/region/territory,
    postal-code,
    country

Reply:
Hi @nayr974 - yep, if we want to talk about a more robust address model, I've got some ideas. Let's chat!

Reply:
Let's address this tomorrow morning, so that I can implement the changes on this and close the branch. Thanks!